.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/aj/venv_directory/my_venv/bin/python"
+}
```

Comment:
It's a best practice to keep IDE/editor artifacts out of your repositories using your .global_gitignore file.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore